src/control/cmd/daos_agent/main.go

```diff
@@ -191,6 +192,11 @@ func parseOpts(args []string, opts *cliOptions, invoker control.Invoker, log *lo
 			defaultConfigPath := path.Join(build.ConfigDir, defaultConfigFile)
 			if _, err := os.Stat(defaultConfigPath); err == nil {
 				cfgPath = defaultConfigPath
+			} else {
+				defaultConfigPath := path.Join("/etc/daos", defaultConfigFile)
+				if _, err := os.Stat(defaultConfigPath); err == nil {
+					cfgPath = defaultConfigPath
+				}
```

Comment:
I'm wondering why this is needed. Isn't the CONF_DIR set to something meaningful (like /etc/daos) at build-time? `build.ConfigDir` should be picking that up.

Reply:
Anyway, we want the developer build and the rpm build to work